exercises/sgf-parsing/canonical-data.json

```diff
@@ -0,0 +1,161 @@
+{
+    "exercise": "sgf-parsing",
+    "version": "1.0.0",
+    "cases": [
+        {
+            "description": "empty input",
+            "property": "parse",
+            "input": {
+                "encoded": ""
+            },
+            "expected": {
+                "error": "tree missing"
+            }
+        },
+        {
+            "description": "tree with no nodes",
+            "property": "parse",
+            "input": {
+                "encoded": "()"
+            },
+            "expected": {
+                "error": "tree with no nodes"
+            }
+        },
+        {
+            "description": "node without tree",
+            "property": "parse",
+            "input": {
+                "encoded": ";"
+            },
+            "expected": {
+                "error": "tree missing"
+            }
+        },
+        {
+            "description": "node without properties",
+            "property": "parse",
+            "input": {
+                "encoded": "(;)"
+            },
+            "expected": {
+                "properties": {}
```

Comment:
interesting that there is no explicit `"children": []` here (it's instead implicit)

Reply:
Good find! Was that on purpose @cmccandless?

Reply:
No, it was not intentional.